crates/prune/types/src/segment.rs

```diff
@@ -15,6 +15,8 @@ pub enum PruneSegment {
     TransactionLookup,
     /// Prune segment responsible for all rows in `Receipts` table.
     Receipts,
+    /// Prune segment responsible for all rows in `AccountTrieChangeSets` table.
+    ChangeSets,
```

Comment:
I'm not sure if it's important to keep naming consistent with the pipeline stages, but if so this should probably be `MerkleChangeSets`. It should at least differentiate itself as being specifically for the trie tables, as the `AccountHistory` and `StorageHistory` segments currently cover the Account/StorageChangeSets tables, respectively